sdk/test/trace/tracer_test.cc

```diff
@@ -1008,6 +1008,26 @@ TEST(Tracer, WithActiveSpan)
     spans = span_data->GetSpans();
     ASSERT_EQ(1, spans.size());
     EXPECT_EQ("span 2", spans.at(0)->GetName());
+    EXPECT_EQ(spans.at(0).get()->GetParentSpanId(), span_first->GetContext().span_id());
+    EXPECT_EQ(spans.at(0).get()->GetTraceId(), span_first->GetContext().trace_id());
+
+    {
+      trace_api::StartSpanOptions options;
+      opentelemetry::context::Context c1;
+      c1             = c1.SetValue("is_root_span", true);
+      options.parent = c1;
+      auto root_span = tracer->StartSpan("span root", options);
```

Comment:
Suggestion:

Add this sequence of code as verbatim example of how to start a root span,
in comments in span_start_options.h

```
trace_api::StartSpanOptions options;
opentelemetry::context::Context root;
root                    = root.SetValue("is_root_span", true);
options.parent = root;
auto root_span = tracer->StartSpan("span root", options);
```

It is not trivial for someone who is not familiar with the API,
so providing code ready to copy and paste will help.

Reply:
Good point. Done.